doc/source/train/examples/intel_gaudi/sd.ipynb

```diff
@@ -0,0 +1,428 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(hpu_sd_finetune)=\n",
```

Comment:
```suggestion
```

I don't think we need this - just refer to `train/examples/intel_gaudi/sd` and sphinx will resolve the reference to this file.